.travis.yml

```diff
@@ -95,14 +95,17 @@ matrix:
         - GCOV=gcov-7
         - CLANG_CXX=clang++-7
         - LLVM_CONFIG=llvm-config-7
+        - COVERAGE=1
```

Comment:
Instead of defining COVERAGE, we can just upload coverage whenever we're using Rust nightly.

Reply:
This way we also merge the coverage we get in the different compiler configurations